CRVResponse/src/MakeCrvRecoPulses.cc

```diff
@@ -11,47 +10,43 @@ namespace
     double const x = xs[0];
     return par[0]*(TMath::Exp(-(x-par[1])/par[2]-TMath::Exp(-(x-par[1])/par[2])));
   }
+  double Gumbel2(double* xs, double* par)
```

Comment:
Do we plan to use Gubel2 or NoFit functions?
What's the default?

Reply:
The default is Gumbel. Gumbel2 can be turned on in the fcl file. The "NoFit parameters" are always extracted: the start time and end time of a pulse, and the time of the maximum ADC value. These additional parameters can be used (via a fcl file option) in the coincidence finder.